### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Pavlo000**
[DEMO LINK](https://Pavlo000.github.io/react_people-table-basics/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+        <Link className="navbar-item" to="/people">
+          People
+        </Link>
+      </nav>
+
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/people" element={<PeopleTable />} />
+        <Route path="*" element={<NotFoundPage />} />
+      </Routes>
```
Comment (jstmpelowycz) [inline src/App.tsx:26]: Screen Shot 2022-08-04 at 11 35 53

File: src/components/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './HomePage.scss';
+
+export const HomePage: React.FC = () => (
+  <h1
+    className="HomePage title"
+  >
```
Comment (jstmpelowycz) [inline src/components/HomePage/HomePage.tsx:7]: inline

File: src/components/PeopleTable/PeopleTable.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      try {
+        const peopleFromServer = await getPeople();
+
+        setPeople(peopleFromServer);
+        setLoading(false);
+      } catch (error) {
+        setLoadingError(true);
+        setLoading(false);
+      }
+    })();
```
Comment (jstmpelowycz) [inline src/components/PeopleTable/PeopleTable.tsx:26]: IIFE :harold:

Comment (jstmpelowycz) [reply src/components/PeopleTable/PeopleTable.tsx:26]: 🔥

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]

**Review by mkyloa — APPROVED** [review]

**Pull request closed without merging**